src/global_mapper/global_mapper_params.h

```diff
@@ -0,0 +1,21 @@
+// Copyright 2017 Massachusetts Institute of Technology
+#pragma once
+
+namespace global_mapper {
+class GlobalMapperParams {
+ public:
+  GlobalMapperParams();
+
+  double voxel_xyz0_[3];
+  double voxel_xyz1_[3];
+  double voxel_meters_per_pixel_[3];
+  double voxel_init_value_;
+  double voxel_min_range_;
+  double voxel_max_range_;
+  double voxel_min_z_abs_;
+  double voxel_max_z_abs_;
+  bool voxel_use_rel_cropping_;
+  double voxel_min_z_rel_;
+  double voxel_max_z_rel_;
```

Comment:
I would make this a struct instead of a class unless you plan on implementing methods for it. Also, you should be able to initialize the member values when you declare them:
```c++
struct GlobalMapperParams {
  double voxel_xyz0 = {0.0, 0.0, 0.0}; 
  ....
  double voxel_init_value = 0.0;
```